app/assets/stylesheets/partials/_one_box.scss

```diff
@@ -1,3 +1,3 @@
 .one-box {
-  height: 100%;
+  // height: 100%;
```

Comment:
Why did you need to comment this out?

Reply:
I must have been experimenting, should not be commented out, will fix.